src/cargo/sources/registry.rs

```diff
@@ -464,8 +464,9 @@ impl<'cfg> RegistrySource<'cfg> {
         // git fetch origin
         let url = self.source_id.url().to_string();
         let refspec = "refs/heads/*:refs/remotes/origin/*";
-        try!(git::fetch(&repo, &url, refspec).chain_error(|| {
-            internal(format!("failed to fetch `{}`", url))
+
+        try!( git::fetch(&repo, &url, refspec, &self.config).chain_error(|| {
```

Comment:
looks like an extra leading space snuck in here and the alignment below is a little off